src/index.ts

```diff
@@ -70,14 +77,11 @@ export class PayIDClaim implements ICruxPayClaim {
     public identitySecrets: string | nameservice.IIdentityClaim | undefined;
     private _getEncryptionKey: () => string;
     private _encryption: typeof encryption.Encryption = encryption.Encryption;
-    private _storage: storage.StorageService = new storage.LocalStorage();
 
     constructor(cruxPayObj: ICruxPayClaim = {} as ICruxPayClaim, options: cruxPayOptions) {
```

Comment:
can rename `cruxPayOptions` to something specific to PayIDClaim like `payIDClaimOptions`.